src/stories/OrderSummary.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import type { Meta, StoryObj } from '@storybook/react';
-import { OrderSummary } from '../components/OrderSummary';
+import OrderSummary from '../components//order/OrderSummary';
```

Comment:
`/`가 하나 더 들어간 것 같아욤!
```suggestion
import OrderSummary from '../components/order/OrderSummary';
```

Reply:
바로 수정하도록 하겠습니다!! 감사합니다!!